app/(landing)/views/offer/token.tsx

```diff
@@ -5,134 +5,189 @@ import Image from 'next/image'
 import clsx from 'clsx'
 import { createContext, useContext } from 'react'
 import Button from '../../../components/button/button'
+import { InternalLink } from '../../../components/links/links'
 import styles from './token.module.scss'
 
 import BtcImage from './image/BTC_logo.png'
 import EthImage from './image/ETH_logo.png'
 import UsdtImage from './image/USDT_logo.png'
+import UsdcImage from './image/USDC_logo.png'
+import BnbImage from './image/BNB_logo.png'
+import DaiImage from './image/DAI_logo.png'
 import { Chip } from './chip'
 
-export type TokenSymbol = 'ETH' | 'BTC' | 'USDT'
+export type TokenSymbol = 'ETH' | 'BTC' | 'USDT' | 'USDC' | 'DAI' | 'BNB'
 
 export interface TokenProps {
   /** Used for displaying symbol and extract stats */
   token: TokenSymbol
   development?: boolean
+  balance?: React.ReactNode
+  href?: string
+  buttonLabel?: string
 }
 
 export const TokenContext = createContext<TokenProps>({ token: 'ETH', development: true })
 export const useToken = () => useContext(TokenContext)
 
-export function Token({ token, children, development }: PropsWithChildren<TokenProps>) {
-  return <div className={clsx(styles.token, styles[token])}>
-        <TokenContext.Provider value={{ token, development }}>
-            <div className={styles.content}>
-                <Logo {...{ token, development }} />
+export function Token({ token, children, development, balance, href, buttonLabel }: PropsWithChildren<TokenProps>) {
+  return (
+    <div className={clsx(styles.token, styles[token])}>
+      <TokenContext.Provider value={{ token, development }}>
+        <div className={styles.content}>
+          <Logo {...{ token, development }} />
 
-                {children}
```

Comment:
Why does PR change indentation? I think we have linter and editorconfig that define exact indendation

Reply:
idk, i use eslint formatter

Reply:
Are you have another config in IDE?

Reply:
no, i have no customs there